test/helpers.js

```diff
@@ -0,0 +1,27 @@
+// retrieve string that matches regex
+var _contains = function (arr, str) {
```

Comment:
The name `_contains` implies only checking and return only a boolean value and nothing else. Let's rename it to `_getFromProcessArgs` and return `null` when nothing can be retrieved.